### ISSUE TITLE
Issue if project or repo name has spaces

### ISSUE THREAD

**Issue by Cancer-zern** [codingsteff/azure-devops-backup-repos#1]
Hello,

I would like to inform you that you can add these fix-solutions if you have a repo or project name with spaces.
check function Backup-Repo and below you can find it.
Just fixes
```
	$url=$url.replace(' ','%20')
        $existProject = Test-Path "$repoName"
        $cmd = "git -C ""$repoName"" pull ""$url"""
        $cmd = "git clone ""$url"" ""$repoName"""
```

Entire fixed "function Backup-Repo"
```
function Backup-Repo($project, $repo) {
    $repoName = $repo.name
    $projectName = $project.name
    $url = "https://$organization@dev.azure.com/$organization/$projectName/_git/$repoName"
    $url=$url.replace(' ','%20')
    Write-Host "   " $repoName
    $existProject = Test-Path "$repoName"
    if ($existProject) {
        # Invoke-Expression "git remote prune origin"
        $cmd = "git -C ""$repoName"" pull ""$url"""
    }
    else {          
        $cmd = "git clone ""$url"" ""$repoName"""
    }
    Write-Host "      " $cmd
    $cmd = Add-Token $cmd $personalAccessToken
    Invoke-Expression $cmd
    Write-Host ""
}
```

**Comment by codingsteff** [comment]
Pull request welcome

**Comment by codingsteff** [comment]
Fixed in #2
